src/client/stories/suggestedRoutes/suggestedRouteCard.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SuggestedRouteCard from "../../components/SuggestedRoutes/SuggestedRouteCard";
+import {
+  withKnobs,
+  text,
+  boolean,
+  number,
+  array,
+  object
+} from "@storybook/addon-knobs";
+
+export function suggestedRouteCard() {
+  return (
+    <SuggestedRouteCard
+      time={text("Time")}
+      price={text("Price")}
+      eta={text("ETA")}
+      sailDays={text("Sail days")}
```

Comment:
Sails days also I think in number

Reply:
yes, but it will be in storyBook to show the demo!

Reply:
True, but when we add propTypes checking this will fail in Storybook. So please change it to number.